spec/ddtrace/tracer_spec.rb

```diff
@@ -253,12 +327,39 @@
       end
 
       context 'when nesting spans' do
-        it 'only the top most span has a runtime ID tag' do
+        it do
```

Comment:
I've noticed that in this file we often use `it do` with quite complex assertions in the block. This is somewhat of an anti-pattern, because the automatic description that rspec generates is just nonsense:

````
$ bundle exec rspec spec/ddtrace/tracer_spec.rb:333
Datadog::Tracer
  #trace
    given a block
      when nesting spans
        is expected to eq "tag_2"
````

I suggest adding a description for the `it` to avoid this; for instance in this case we could use `it 'nests children spans created inside a parent span'` or something similar?

Reply:
I don't recall deleting this comment, but yeah, I agree that making sure the RSpec output is clear and descriptive should always be the case. I'll address it.